experiments/experiments.go

```diff
@@ -334,3 +334,24 @@ func isSimpleExperiment(experimentType string) bool {
 	}
 	return false
 }
+
+// ExperimentEvent is the playload used by Expose to log whether a user has
+// been exposed to an experimental treatment.
+type ExperimentEvent struct {
+	ID              uuid.UUID
+	CorrelationID   uuid.UUID
+	DeviceID        uuid.UUID
+	Experiment      *ExperimentConfig
+	VariantName     string
+	UserID          string
+	LoggedIn        bool
+	ClientTimestamp int64
+	CookieCreatedAt int64
```

Comment:
For `ClientTimestamp` and `CookieCreatedAt`, can we use a more concrete type from `timebp` to indicate the unit? Or better yet, just use `time.Time` and convert that into required type of int in `Expose` function?

Reply:
Dang! I always miss those, thanks! :)